ui-ngx/src/app/modules/home/components/dashboard-page/dashboard-page.component.ts

```diff
@@ -564,6 +564,11 @@ export class DashboardPageComponent extends PageComponent implements IDashboardC
     });
   }
 
+  public showUserMenu(): boolean {
+    return !this.isPublicUser() && this.forceFullscreen && (this.authState.userDetails.additionalInfo?.defaultDashboardId ?
+      true : this.authState.userDetails.additionalInfo?.homeDashboardHideToolbar);
+  }
```

Comment:
The attached images show that the platform is not switching to full-screen mode. This implies that the change in forceFullscreen has not been correctly calculated in your case.